### PR TITLE
[MTDSA-3084] POC for Play error handling

### PR DESCRIPTION
**Pull request by MarkAKelly**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -38,7 +38,7 @@ play.http.filters = "uk.gov.hmrc.play.bootstrap.filters.MicroserviceFilters"
 
 # Json error handler
 play.http.errorHandler = "uk.gov.hmrc.play.bootstrap.http.JsonErrorHandler"
```
Comment (ma3574) [inline conf/application.conf]: Shouldn't this line be removed 😱

Comment (MarkAKelly) [reply conf/application.conf]: Potentially yes.

Comment (ma3574) [reply conf/application.conf]: mmmmmyesssssss

Comment (MarkAKelly) [reply conf/application.conf]: I accidentally made it into a PR.

**Pull request closed without merging**